craft_providers/lxd/remotes.py

```diff
@@ -58,6 +58,40 @@ class RemoteImage:
     remote_protocol: ProtocolType
     is_stable: bool
 
+    def add_remote(self, lxc: LXC = LXC()) -> None:
```

Comment:
out of curiosity, what's the semantics of this default value? When would we use that?

Reply:
It's a great question and I don't have a good answer.

I've kept it because it was already there in the `configure_buildd_image_remote(lxc: LXC = LXC())` function.

I think this would be a good opportunity to remove the default value.  Done!